holistic_bias/src/hb_counts.py

```diff
@@ -0,0 +1,168 @@
+
+import re
+import sys
+import typing as tp
+
+from collections import Counter
+from pathlib import Path
+from tqdm import tqdm
+
+import fasttext
+
+from holistic_bias.src.sentences import HolisticBiasSentenceGenerator
+
+
+
+class CountHolisticBias(object):
+    """
+    This is the core class for computing Holistic Biases distribution. 
+    It does the counting based on Holistic_Bias list of nouns and noun phrases, return counters
+
+    Based on https://github.com/fairinternal/seamless_common/blob/hb_count/stopes/eval/holistic_bias/callback.py
```

Comment:
External users won't have access to this path - might be good to remove this line